src/cli.zig

```diff
@@ -1988,27 +2022,31 @@ pub const Command = struct {
                 },
                 Command.Tag.CreateCommand => {
                     const intro_text =
-                        \\<b>Usage<r>:
-                        \\  <b><green>bun create<r> <blue>\<template\><r> <cyan>[...flags]<r> <blue>[dest]<r>
-                        \\  <b><green>bun create<r> <blue>\<username/repo\><r> <cyan>[...flags]<r> <blue>[dest]<r>
-                        \\
-                        \\<b>Environment variables:<r>
-                        \\  <cyan>GITHUB_ACCESS_TOKEN<r>      <d>Supply a token to download code from GitHub with a higher rate limit<r>
-                        \\  <cyan>GITHUB_API_DOMAIN<r>        <d>Configure custom/enterprise GitHub domain. Default "api.github.com".<r>
-                        \\  <cyan>NPM_CLIENT<r>               <d>Absolute path to the npm client executable<r>
+                        \\<b>Usage<r>: <b><green>bun create<r> <cyan>\<template\><r> [\<destination\>]
+                        \\<b>Alias: <b>bun c<r>
+                        \\Create a new bun project using a create-\<template\> npm package, GitHub repo, or a local template
                     ;
 
                     const outro_text =
-                        \\If given a GitHub repository name, Bun will download it and use it as a template,
-                        \\otherwise it will run <b><magenta>bunx create-\<template\><r> with the given arguments.
+                        \\<b>Examples:<r>
+                        \\  <d>From npm<r>
+                        \\  <b><green>bun create remix<r>
+                        \\
+                        \\  <d>From GitHub<r>
+                        \\  <b><green>bun create \<user\>/\<repo\><r>
                         \\
-                        \\Learn more about creating new projects: <magenta>https://bun.sh/docs/cli/bun-create<r>
+                        \\  <d>From local template<r>
+                        \\  <b><green>bun create $HOME/.bun-create/\<name\><r>
```

Comment:
wait does this work?

Reply:
I took this elements from the documentation but I don't tested. If the documentation is correct I suppose yes it's work